enterprise/frontend/src/embedding-sdk-ee/tests/auth-flow/jwt.unit.spec.tsx

```diff
@@ -104,6 +121,22 @@ describe("Auth Flow - JWT", () => {
     );
   });
 
+  it("should skip the initial /auth/sso request when jwtProviderUri is provided", async () => {
+    const authConfig = defineMetabaseAuthConfig({
+      metabaseInstanceUrl: MOCK_INSTANCE_URL,
+      preferredAuthMethod: "jwt",
```

Comment:
the test only mentions `jwtProviderUri` but not `preferredAuthMethod`. I also feel like API wise, we shouldn't even need `preferredAuthMethod` if `jwtProviderUri` is provided. I mean you could specify `preferredAuthMethod: "saml"` and `jwtProviderUri`, but what does that mean?

Also, consider this, I don't think `preferredAuthMethod` has a default value, that means whenever users want to specify `jwtProviderUri` they need to also specify `preferredAuthMethod: "jwt"` which could be a bit cumnbersome. Correct me if I'm wrong on that.